oak_containers/proto/interfaces.proto

```diff
@@ -79,6 +97,15 @@ service Launcher {
   rpc NotifyAppReady(google.protobuf.Empty) returns (google.protobuf.Empty) {}
 }
 
+// Defines the service exposed by the orchestrator, that can be invoked by the application.
+service ControlPlane {
+  // Request a new enclave encryption key as part of Key Provisioning.
+  rpc GetEncryptionKey(GetEncryptionKeyRequest) returns (GetEncryptionKeyResponse) {}
+
+  // Replace old enclave encryption key with a new one as part of Key Provisioning.
```

Comment:
I think we agreed that we will not replace the old key, just add the new alongside the old one.

Reply:
Updated the comment